Plugins/Flow.Launcher.Plugin.Program/Main.cs

```diff
@@ -72,22 +72,17 @@ public List<Result> Query(Query query)
             Win32[] win32;
             UWP.Application[] uwps;
 
-            lock (IndexLock)
-            { // just take the reference inside the lock to eliminate query time issues.
-                win32 = _win32s;
-                uwps = _uwps;
-            }
```

Comment:
Bao has removed the lock in Wox, so I think removing it won't be an issue in Flow, too.

Reply:
I dont suppose those two will be updated often unless a reindex is run. Can the user reindex and also query at the same time?

Reply:
I would consider even if they query and reindex at the same time, it won't affect each other since index only change the memory index after finishing the whole index process (which is time consuming), but assigning values should not be an issue. On the time that Flow is reindexing, the variable won't change until it finish.

Reply:
I have provide a small test and it seems that won't fine of querying when reindexing.